lib/maintainers.nix

```diff
@@ -552,6 +552,7 @@
   schristo = "Scott Christopher <schristopher@konputa.com>";
   scolobb = "Sergiu Ivanov <sivanov@colimite.fr>";
   sdll = "Sasha Illarionov <sasha.delly@gmail.com>";
+  seanz = "SeanZicari <sean.zicari@gmail.com>";
```

Comment:
I'm sorry, I didn't mean your name, I meant the `seanz` part (see the top of the file for clarification).